cli/src/git_util.rs

```diff
@@ -172,17 +173,122 @@ pub fn with_remote_git_callbacks<T>(
     f(callbacks)
 }
 
-pub fn print_git_import_stats(ui: &mut Ui, stats: &GitImportStats) -> Result<(), CommandError> {
+pub fn print_git_import_stats(
+    ui: &mut Ui,
+    stats: &GitImportStats,
+    query_ref_stats: Option<&ReadonlyRepo>,
+) -> Result<(), CommandError> {
```

Comment:
nit: better to pass `repo` and `show_ref_stats` separately because there may be other things that requires `repo`.

`(Ui, dyn Repo, GitImportStats, bool)`

Reply:
resolved